src/test/java/org/apache/commons/lang3/StringUtilsTest.java

```diff
@@ -2563,6 +2563,24 @@ public void testIsAllUpperCase() {
         assertFalse(StringUtils.isAllUpperCase("A/C"));
     }
 
+    /**
+     * Test for {@link StringUtils#isMixedCase(CharSequence)}.
+     */
+    @Test
+    public void testIsMixedCase() {
+        assertFalse(StringUtils.isMixedCase(null));
+        assertFalse(StringUtils.isMixedCase(StringUtils.EMPTY));
+        assertFalse(StringUtils.isMixedCase(" "));
+        assertTrue(StringUtils.isMixedCase("aBc"));
+        assertFalse(StringUtils.isMixedCase("abc"));
+        assertFalse(StringUtils.isMixedCase("ABC"));
+        assertFalse(StringUtils.isMixedCase("aBc "));
+        assertFalse(StringUtils.isMixedCase("A c"));
+        assertFalse(StringUtils.isMixedCase("aBc\n"));
+        assertFalse(StringUtils.isMixedCase("A1c"));
+        assertFalse(StringUtils.isMixedCase("a/C"));
```

Comment:
Done, and I added a quick return case to skip going into the loop unnecessarily because a char sequence of length 1 cannot have both an upper and lowercase.